contrib/authors_update.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eu
+
+range="${1?Provide commit range like orig/v1.17..orig/v1.18}"
```

Comment:
add a check that the range is valid? (not empty)

Reply:
fixed with git log